castle/cms/services/google/analytics.py

```diff
@@ -9,20 +9,27 @@ def get_ga_service(ga_scope=['https://www.googleapis.com/auth/analytics.readonly
     api_email = registry.get('castle.google_api_email', None)
     api_key = registry.get('castle.google_api_service_key_file', None)
     ga_id = registry.get('castle.google_analytics_id', None)
+    ua_id = registry.get('castle.universal_analytics_id', None)
 
-    if not api_key or not api_email or not ga_id:
+    if not api_key or not api_email or not ga_id and not ua_id:
```

Comment:
while this is correct, i would suggest making it a bit clearer which operators should be evaluated first: `if not api_key or not api_email or (not ga_id and not ua_id):` I think it's easier to read that way. But, take or leave that suggestion